acceptance/testdata/complex.386.yaml

```diff
@@ -19,7 +19,7 @@ homepage: "https://foobar.org"
 license: "MIT"
 files:
   ../testdata/fake: "/usr/local/bin/fake"
-  ./testdata/folder/**/*: "/usr/share/whatever/folder/"
```

Comment:
I'm not sure which is wrong, or if any of them are, but results are different now 🤔